lib/active_model/serializer.rb

```diff
@@ -122,8 +122,12 @@ def initialize(object, options = {})
       end
     end
 
+    def object_model_name
+      object.class.model_name
+    end
+
```

Comment:
I like the encapsulation.. would like to think a bit about adding another method, and also the name of this method.. or should we just use `serializer.json_key` and let you overwrite that?